ruleset/rules/0630-nextcloud_rules.xml

```diff
@@ -1,175 +1,156 @@
 <!--
-  -  NextCloud rules
-  -  Author: banditopazzo.
-  -  Updated by Wazuh, Inc.
-  -  Copyright (C) 2015-2021, Wazuh Inc.
-  -  This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+  Copyright (C) 2015-2021, Wazuh Inc.
 -->
 
 <!--
-- It's needed to enable audit in Nextcloud if you want full log
-- It's needed to paste the following in /var/ossec/etc/ossec.conf for the decoder to work properly:
-<localfile>
-	<location>ABSOLUTE_PATH_TO_YOUR_NEXTCLOUD_.json_LOG_FILE</location>
-	<log_format>json</log_format>
-	<label key="@source">NextCloud</label>
-</localfile>
-- OPTIONAL
-<localfile>
-	<location>ABSOLUTE_PATH_TO_YOUR_AUDIT_.json_LOG_FILE</location>
-	<log_format>json</log_format>
-	<label key="@source">NextCloud</label>
-</localfile>
+  Enable audit in Nextcloud if you want full log
+  Paste the following in /var/ossec/etc/ossec.conf for the decoder to work properly:
+  <localfile>
+    <location>ABSOLUTE_PATH_TO_YOUR_NEXTCLOUD_.json_LOG_FILE</location>
+    <log_format>json</log_format>
+    <label key="@source">NextCloud</label>
+  </localfile>
+  OPTIONAL
```

Comment:
This line should be redacted better. No need for uppercase. There is a need for a better explanation on why that is optional. There is also a need for an explanation to know if this block is intended to replace the one above or should be appended below.

Reply:
This is legacy comment, I have never used NextCloud, should investigate "from scratch"

Reply:
What about if instead of OPTIONAL we use "To get the full log message from audit:".
Not sure what are the ABSOLUTE variables referring to

Reply:
I am guessing having both enabled would generate double detections as rules 88200 and 88201 ingest logs from audit nextcloud and raw nextcloud

Reply:
From the way the rules are written it looks like the rules will trigger with either method, will improve the description based on that